src/components/PlaybackApp/controls.js

```diff
@@ -30,14 +30,12 @@ export const playbackControlStart = function(playback, stream) {
           .then(() => {
             music.player.pause()
               .then(() => {
-                setTimeout(() => {
-                  const delayArr = getPositionMilliseconds(stream, stream.startedAt),
-                        delayPositionMilliseconds = delayArr[0];
-                  music.player.seekToTime(delayPositionMilliseconds / 1000)
-                    .then(() => {
-                      music.player.play();
-                    });
-                }, 250);
+                const delayArr = getPositionMilliseconds(stream, stream.startedAt),
```

Comment:
MusicKit JS only really works on Safari since it's buggy in other browsers. This is a little playback optimization for Safari.